h2-storage/src/main/java/com/cgi/hexagon/h2storage/order/H2OrderStorage.java

```diff
@@ -61,6 +63,12 @@ public List<Order> findAll() {
         orderRepository.findAll().forEach(order -> result.add(mapper.fromEntity(order)));
         return result;
     }
+
+    public List<Order> findByStatus(Status status) {
+        List<Order> result = new ArrayList<>();
+        orderRepository.findByStatus(status).forEach(order -> result.add(mapper.fromEntity(order)));
```

Comment:
Use higher order function instead

Reply:
I mean, use map funtion like in the findAllByShopOrderId method

Reply:
findAllByStatus a littlebit better name when the return is a List